src/main/scala/com/typesafe/sbt/packager/graalvmnativeimage/GraalVMNativeImagePlugin.scala

```diff
@@ -87,18 +96,20 @@ object GraalVMNativeImagePlugin extends AutoPlugin {
 
   private def buildLocal(targetDirectory: File,
                          binaryName: String,
+                         nativeImageCommand: String,
                          className: String,
                          classpathJars: Seq[File],
                          extraOptions: Seq[String],
                          log: ProcessLogger): File = {
     targetDirectory.mkdirs()
     val command = {
       val nativeImageArguments = {
-        val classpath = classpathJars.mkString(":")
+        val classpath = classpathJars.mkString(System.getProperty("path.separator"))
```

Comment:
We don't need to hardcode the property name:
```suggestion
        val classpath = classpathJars.mkString(java.io.File.pathSeparator)
```